core/src/main/java/org/apache/iceberg/hadoop/ConfigProperties.java

```diff
@@ -26,4 +26,8 @@ private ConfigProperties() {
 
   public static final String ENGINE_HIVE_ENABLED = "iceberg.engine.hive.enabled";
   public static final String KEEP_HIVE_STATS = "iceberg.hive.keep.stats";
+  public static final String CACHE_MAX_CONTENT_LENGTH = "iceberg.cache.max-content-length";
```

Comment:
This should not be configured through Hadoop. Instead, it should be configured with catalog properties.

Reply:
Done